src/database/models/team_item.rs

```diff
@@ -78,17 +83,68 @@ impl TeamBuilder {
 pub struct Team {
     /// The id of the team
     pub id: TeamId,
+    // The association of the team: either a project or an organization
+    pub association: TeamAssociationId,
+}
+
+#[derive(Deserialize, Serialize, Clone, Debug, Copy)]
+pub enum TeamAssociationId {
+    Project(ProjectId),
+    Organization(OrganizationId),
+}
```

Comment:
think this should be removed from the team/is not needed

Reply:
Kept the structure as its useful, but no longer in the db as discussed on disc